docs/src/surfaces/rail/rail.jsx

```diff
@@ -64,8 +64,6 @@ export default class ElementsRail extends React.PureComponent {
 
         return (
             <Main page="rail">
-                <TitleBar title="Rail" />
```

Comment:
`<TitleBar />` was removed in here but not in line 6 (importing from `react-cm-ui`).

Reply:
Fixed.